b2sdk/v0/sync.py

```diff
@@ -0,0 +1,189 @@
+######################################################################
+#
+# File: b2sdk/v0/sync.py
+#
+# Copyright 2019 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+import logging
+
+from b2sdk.v1.exception import CommandError
+from b2sdk.v1.exception import DestFileNewer as DestFileNewerV1
+from b2sdk.v1 import trace_call
+from .exception import DestFileNewer
+from b2sdk.v1.exception import InvalidArgument, IncompleteSync
+from b2sdk.v1 import AbstractFileSyncPolicy
+from b2sdk.v1 import DEFAULT_SCAN_MANAGER
+from b2sdk.v1 import SyncReport
+from b2sdk.v1 import Synchronizer as SynchronizerV1
+
+logger = logging.getLogger(__name__)
+
+
+class Synchronizer(SynchronizerV1):
```

Comment:
How about a class doc comment saying what this adapter is doing?

Reply:
new style docs are tbd awaiting general interface approval. Now that we have it, I'll add docstrings, modify `doc/quick_start.rst` etc